### ISSUE TITLE
ios currentUser phoneNumber is null after app restart/onAuthStateChanged() login

### ISSUE THREAD

**Issue by minutuslausus** [EddyVerbruggen/nativescript-plugin-firebase#465]
On ios If I register a user with phone auth then after the login I have the phoneNumber field on the userobject with my phone number. 

But if I close the app and restart is the login is automatic through the onAuthStateChanged() method, but I have phoneNumber: null in this case. 

CONSOLE LOG file:///app/shared/services/firebase.service.js:90:20: fbService: onAuthChange {"loggedIn":true,"user":{"uid":"pocfGfrYSxd6zFKf2g1mtTTbYPG3","anonymous":false,"providers":[{"id":"phone"}],"profileImageURL":null,"email":null,"emailVerified":false,"name":null,"phoneNumber":"+36...","..."}}
CONSOLE LOG file:///app/shared/services/firebase.service.js:90:20: fbService: onAuthChange {"loggedIn":true,"user":{"uid":"pocfGfrYSxd6zFKf2g1mtTTbYPG3","anonymous":false,"providers":[{"id":"phone"}],"profileImageURL":null,"email":null,"emailVerified":false,"name":null,"phoneNumber":null,"refreshToken":"...."}}

it works on android.

		"nativescript-angular": "~4.2.0",
		"nativescript-plugin-firebase": "^4.0.5",

**Comment by EddyVerbruggen** [comment]
Yep, I can confirm this is happening and it originates in the Firebase iOS SDK. Perhaps you can take a look at their doc or code (it's mostly open source these days). Heck, it may even have been fixed already and we only need to bump the SDK version..
